code/modules/photography/camera/silicon_camera.dm

```diff
@@ -89,3 +89,13 @@
 	C.toner -= printcost  //All fun allowed.
 	visible_message(span_notice("[C.name] spits out a photograph from a narrow slot on its chassis."))
 	to_chat(usr, span_notice("You print a photograph."))
+
+/obj/item/camera/siliconcam/proc/paiprint(mob/user)
+	var/mob/living/silicon/pai/C = loc
```

Comment:
the fuck does C mean

Reply:
C is just generic variable used for the "visable message". Basically originally was a copy-pasta of borgoprint but I retooled it to use the default camera function instead.

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#dont-use-abbreviations

Reply:
fixed.